src/stories/Library/Buttons/button-share/button-share.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactComponent as LinkSvg } from "../../../../public/icons/collection/Link.svg";
+import { ReactComponent as FacebookSvg } from "../../../../public/icons/social/icon-social-facebook.svg";
+
+export type ButtonShareProps = {
+  href: string;
+  textFacebook: string;
+  textCopy: string;
+};
+
+export const ButtonShare = ({
+  href,
+  textFacebook,
+  textCopy,
+}: ButtonShareProps) => {
+  return (
+    <div className="button-share">
+      <div className="button-share button-share--fixed">
+        <a
```

Comment:
We have a `<LinkButton />` component in the repository. Utilize that here, please. (if it doesn't fit for some reason, we also have a `<Links />` component for links (<a/> tags) (not sure why it's defined in plural 🤷)

Reply:
`<Links />` takes a string as link text, but I need to render an SVG, that it why I cannot use the `<Links />` component